qiskit/circuit/library/blueprintcircuit.py

```diff
@@ -131,10 +131,10 @@ def __len__(self):
     def __getitem__(self, item):
         return self.data[item]
 
-    def size(self):
+    def size(self, *args, **kwargs):
         if self._data is None:
             self._build()
-        return super().size()
+        return super().size(*args, **kwargs)
```

Comment:
i don't know if this is the correct way of doing this

Reply:
I think this is fine.